totalRP3/Locales/Locale.lua

```diff
@@ -489,6 +491,16 @@ Class: 50 characters|r]],
 	CO_REGISTER_AUTO_PURGE_0 = "Disable purge",
 	CO_REGISTER_AUTO_PURGE_1 = "After %s day(s)",
 	CO_CURSOR_TITLE = "Cursor interactions",
+	CO_RELATIONS = "Relations settings",
+	CO_RELATIONS_NEW = "Create new relation",
+	CO_RELATIONS_NEW_TITLE = "Enter the name of the new relation",
+	CO_RELATIONS_NEW_ERROR = "You must enter a name for the new relation.",
+	CO_RELATIONS_NEW_ERROR_ID = "This relation already exists.",
+	CO_RELATIONS_DELETE = "Delete relation",
+	CO_RELATIONS_DELETE_WARNING = "Are you sure you want to delete the relation |cff00ff00%s|r?",
```

Comment:
I'd rather color codes not be in localized strings - both for reasons of it being a pain for people to localize around, and also because it makes future changes to colours elsewhere very hard without invalidating a bunch of strings.

And yes - I know half the strings in the file violate that rule, but in this case it's trivially avoided as the subject is being formatted in anyway - so the colour can just be formatted in with it.

Reply:
Not sure why this ended up in here, other than me not double checking generated code.